### DISCUSSION TITLE
qBittorrent - Modifying alltime stats

### DISCUSSION THREAD

**Discussion by landaisbenj** [alexbelgium/hassio-addons#1068] [Q&A] [answered]
Hello, did you know if it's possible to adjust "alltime stats" ? My tracker's ratio is about 673.09Go - 400.76Go and qbittorrent alltime stat is just 420Go - 242Go... Maybe there is a solution to modify alltime stat?

**Comment by alexbelgium** [comment, marked as answer]
Hi, no idea sorry... I don't even know if it comes from qbittorrent or vuetorrent... I checked on internet and it seems a global issue : https://www.reddit.com/r/qBittorrent/comments/15ziz1w/why_is_alltime_stats_so_inaccurate_and_is_it/ ; linking to https://github.com/qbittorrent/qBittorrent/issues/19173.

So globally it won't be possible to help from this addon repo ; the only possible solution would come from the qbittorrent github linked above. Hope for you you'll find a way !

**Comment by landaisbenj** [comment]
Thanks for your time :)
